recipe/meta.yaml

```
@@ -1,208 +1,211 @@
+  - url: {{ github }}  # [linux64 or aarch64]
+    sha256: a4b0965e7193515930542cc558d922e24c1732d8240efde3e622e84d6c4416fe  # [linux64 or aarch64]
 
 build:
   number: {{ build_num }}
-  skip: true  # [not linux or cuda_compiler_version not in ("11.8", "12.0")]
+  skip: true  # [not cuda_compiler_version in ("11.8", "12.0", "12.6")]
+  skip: true  # [win]
+  skip: true  # [osx]
 
-outputs:
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ compiler("cuda") }}
 
+outputs:
   - name: cuquantum
-    version: {{ version }}
+    version: {{ ".".join(sdk_version[:3]) }}
     build:
       number: {{ build_num }}
     requirements:
       build:
       host:
       run:
-        - {{ pin_subpackage('custatevec', max_pin='x.x.x') }}
-        - {{ pin_subpackage('cutensornet', max_pin='x.x.x') }}
+        - {{ pin_subpackage("cudensitymat", max_pin="x.x.x") }}
+        - {{ pin_subpackage("cutensornet", max_pin="x.x.x") }}
+        - {{ pin_subpackage("custatevec", max_pin="x.x.x") }}
     test:
       commands:
-        - echo "ok!"
-
-  - name: custatevec
-    version: {{ cusv_version }}
+        - echo "cuquantum metapackage {{ sdk_version_string }}"
+  - name: cudensitymat
+    version: 0.0.5
     build:
       number: {{ build_num }}
-      script:
-        - mkdir -p $PREFIX/include                                            # [linux]
-        - mv include/custatevec.h $PREFIX/include/                            # [linux]
-        - mkdir -p $PREFIX/lib                                                # [linux]
-        - mv lib/libcustatevec.so* $PREFIX/lib/                               # [linux]
+      string: "{{ mpi_prefix }}_h{{ PKG_HASH }}_{{ mpi_suffix }}"
+      script: "${RECIPE_DIR}/cudensitymat/cudm-install"
       run_exports:
-        - {{ pin_subpackage('custatevec', max_pin='x') }}
+        - {{ pin_subpackage("cudensitymat", max_pin="x") }} {{ mpi_prefix }}_*
       ignore_run_exports_from:
-        - {{ compiler('cuda') }}
-      missing_dso_whitelist:
-        - '*/libcublas.so*'    # [(cuda_compiler_version or "").startswith("11")]
-        - '*/libcublasLt.so*'  # [(cuda_compiler_version or "").startswith("11")]
-      post-link: post-link  # this is supported by conda-build, but undocumented
+        - {{ compiler("cuda") }}
+        - libcublas  # [cuda_compiler_version in ("12.0", "12.6")]
+      post-link: post-link
     requirements:
       build:
         - {{ stdlib('c') }}
-        - {{ compiler('c') }}
-        - {{ compiler('cxx') }}
-        - {{ compiler('cuda') }}
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - {{ compiler("cuda") }}
       host:
-        - libcublas  # [(cuda_compiler_version or "").startswith("12")]
-        - cuda-version {{ cuda_compiler_version }}
+        - cutensor >=2.0.2,<3
+        - cuda-version={{ cuda_compiler_version }}  # [cuda_compiler_version in ("11.8", "12.0", "12.6")]
```

```suggestion
        - cuda-version={{ cuda_compiler_version }}
```
Selector not needed since there are only CUDA variants.